pkg/gadgettracermanager/containerutils/crio/crio.go

```diff
@@ -75,20 +76,31 @@ func (c *CrioClient) PidFromContainerId(containerID string) (int, error) {
 		Verbose:     true,
 	}
 
-	status, err := c.client.ContainerStatus(context.Background(), request)
+	res, err := c.client.ContainerStatus(context.Background(), request)
 	if err != nil {
 		return -1, err
 	}
 
-	pidStr, ok := status.Info["pid"]
+	info, ok := res.GetInfo()["info"]
```

Comment:
What about keeping both behaviours in case somebody is using cri-o < 1.18.0? We can try to check if "pid" is there, if yes then parse it (current implementation), otherwise try to parse "info" (as done in this PR).

Reply:
Done